docs/dev/pos/gui.rst

```diff
@@ -0,0 +1,37 @@
+JS access and inheritance
+=========================
+
+action_button
+-------------
+
+Here you will find explanation of how to get/inherit action_button POS objects.
+
+For example we got JS object::
+
+    odoo.define('pos_reprint.pos_reprint', function (require) {
+    ...
+    screens.define_action_button({
+        'name': 'reprint',
+        'widget': ReprintButton,
+        'condition': function()
+
+Owner object (pos_reprint.pos_reprint) do not exports (returns) any data.
+
+In order to get access (inherit) to that object we need get instance of it using gui.
+To make clear what this is like look up example where guests number button renderings::
+
+    this.gui.screen_instances['products'].action_buttons['guests'].renderElement();
+
+Here *this.gui.screen_instances["products"].action_buttons['guests'].prototype* is undefined. So using of
+extend or include is impossible.
```

Comment:
While you can make call and even replace function with new one, you are not able to make inheritance via ``extend`` or ``include`` functions. It's because we cannot reach Class and only get access to instance of that class